modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt

```diff
@@ -68,8 +71,28 @@ import kotlin.test.assertTrue
 @Sdk("Java/Kotlin")
 class InstructionsTest : IrohaTest<Iroha2Client>(
     account = ALICE_ACCOUNT_ID,
-    keyPair = ALICE_KEYPAIR,
+    keyPair = ALICE_KEYPAIR
 ) {
+    @Test
+    @Disabled // EXAMPLE
+    @WithIrohaManual("http://localhost:8080", "http://localhost:8180")
```

Comment:
I'm thinking how about passing a file path to docker compose and start that inside of the annotation 🤩

Reply:
please create a separate task in the backlog for this suggestion